spacepy/pybats/__init__.py

```diff
@@ -624,10 +624,10 @@ def _scan_bin_header(f, endchar, inttype, floattype):
     # Stash relevant values into the "info" dict:
     vals = readarray(f, dtype=header_fields_dtype, inttype=inttype)[0]
     for v, x in zip(['iter', 'runtime', 'ndim', 'nparam', 'nvar'], vals):
-        info[v] = x
+        info[v] = np.int64(x)
 
     # Dimensionality may be negative to indicate non-uniform grid:
-    info['ndim'] = abs(info['ndim'])
+    info['ndim'] = np.abs(info['ndim'], dtype=np.int64)
```

Comment:
This doesn't need the dtype cast...if it's an int64 coming in, it will be one coming out. In fact just plain `abs` works but making it explicitly numpy seems reasonable. Casting to int64 implies it isn't one coming in.

Reply:
Agreed; changed.